include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -159,7 +159,7 @@ struct GemmKernel
 
     CK_TILE_HOST static bool IsSupportedArgument(const GemmKernelArgs& kargs)
     {
-        if constexpr(EpiloguePipeline::GetVectorSizeC() % 2 != 0 &&
+        if constexpr(EpiloguePipeline::template GetVectorSizeC<CDataType>() % 2 != 0 &&
```

Comment:
Why this is needed ?? You actually pass exactly same type as already is in EpiloguePipeline
https://github.com/ROCm/composable_kernel/blob/a4c61a8b46dd4d72f6273caf2fbbc6921920f568/include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp#L72